Lesson05/HW05.SchoolControls/Program.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+
+namespace HW05.SchoolControls
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            
+            while (true)
+            {
+                Console.WriteLine("Введите номер контрольного теста");
```

Comment:
It is desirable to write the internal text in English.

Reply:
done